extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/templates/configmap.yaml

```diff
@@ -2,28 +2,41 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: metastore-service-config
+  labels:
+    aissemble-configuration-store: enabled
```

Comment:
C: This needs to be templatized and set via `values` so it can be changed by downstream projects.